consts.go

```diff
@@ -14,4 +14,6 @@ const (
 	MEAL_LIST = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_list.aspx"
 	// 학식 게시판 주소
 	MEAL_BOARD = SKHU_URL + "uni_zelkova/uni_zelkova_4_3_view.aspx?idx="
+	// 학사일정 주소
+	SCHEDULES_URL = SKHU_URL + "calendar/calendar_list_1.aspx"
```

Comment:
Codacy found an issue: [don't use ALL_CAPS in Go names; use CamelCase](https://app.codacy.com/gh/s-owl/skhuinfo-api/pullRequest?prid=6091219)